tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/e2e-upgrade-test.go

```diff
@@ -49,7 +49,31 @@ var _ = Describe(
 			})
 		})
 
-		AfterAll(func() {
+		It("Upgrade end to end", reportxml.ID("68954"), func() {
+			By("Create Upgrade IBGU", func() {
+				newIbguBuilder := ibgu.NewIbguBuilder(cnfinittools.TargetHubAPIClient,
+					tsparams.IbguName, tsparams.IbguNamespace).
+					WithClusterLabelSelectors(tsparams.ClusterLabelSelector).
+					WithSeedImageRef(cnfinittools.CNFConfig.IbguSeedImage, cnfinittools.CNFConfig.IbguSeedImageVersion).
+					WithOadpContent(cnfinittools.CNFConfig.IbguOadpCmName, cnfinittools.CNFConfig.IbguOadpCmNamespace).
+					WithPlan([]string{"Prep", "Upgrade"}, 5, 30)
+
+				newIbguBuilder, err := newIbguBuilder.Create()
+				Expect(err).ToNot(HaveOccurred(), "Failed to create upgrade Ibgu.")
+
+				_, err = newIbguBuilder.WaitUntilComplete(30 * time.Minute)
+				Expect(err).NotTo(HaveOccurred(), "Prep and Upgrade IBGU did not complete in time.")
+
+			})
+
+			By("Saving target sno cluster info after upgrade", func() {
+				err := cnfclusterinfo.PostUpgradeClusterInfo.SaveClusterInfo()
+				Expect(err).ToNot(HaveOccurred(), "Failed to collect and save target sno cluster info after upgrade")
+			})
+
+		})
+
+		It("Rollback successful upgrade", reportxml.ID("69058"), func() {
```

Comment:
Should skip this test if upgrade test failed based on the previous flag.

Reply:
Added skip test if !upgradeSuccessful